addusers.py

```diff
@@ -240,6 +242,7 @@ def parse_rows(rows, select_user=None):
                                    "Incomplete: {}").format(entry[4])))
             continue
         try:
+            entry[3] = datetime.datetime.now().strftime("%m/%d/%Y %H:%M:%S")
```

Comment:
This would replace the original timestamp, but we don't want to lose that information - the processing time should be stored in an additional column called "Processed" or something like that.